src/WCSimDetectorConfigs.cc

```diff
@@ -29,13 +29,13 @@ void WCSimDetectorConstruction::SetSuperKGeometry()
   WCPMTRadius = PMT->GetRadius();
   WCIDDiameter          = 33.6815*m; //16.900*2*cos(2*pi*rad/75)*m; //inner detector diameter
   WCIDHeight            = 36.200*m; //"" "" height
-  WCBarrelPMTOffset     = 0.0715*m; //offset from vertical
+  WCBarrelPMTOffset     = WCPMTRadius; //offset from vertical
```

Comment:
I did not check whether the number of PMTs are different though, which could change the impact on physics. According to my figure, those two PMTs are not allowed to be there, but I'm not sure whether the PMTs on the Cap are reordened or whether there will be less PMTs with the geometry overlap fix.
I agree that this is strange and should also be understood where the numbers come from. I thought a different definition of WCBarrelPMTOffset could cause this? 
Also, isn't there a database with the exact xyz-location of the SK PMTs, which will then be the ideal comparison with how they are programmed in WCSim?